### ISSUE TITLE
2 webhooks for same category

### ISSUE THREAD

**Issue by kansas07** [pajlads/DinkPlugin#480]
### Checklist

- [X] I've searched the [issues and pull requests](https://github.com/pajlads/Dinkplugin/issues?q=) for similar looking suggestions.
- [X] I've checked the `Unreleased` section of the [changelog](https://github.com/pajlads/DinkPlugin/blob/master/CHANGELOG.md) for newly added features that sound like my suggestion.

### Describe your Suggestion

I'm not sure if this can be done, but I want to send to 2 different webhooks 2 different types of loots (allowlist and value) not send both to both webhooks

**Comment by iProdigy** [comment]
Webhook-specific options would require us to create a custom UI panel (and substantial design effort).

In the meantime you have a couple options:

1. Use another plugin in conjunction with Dink dedicated to the separate webhook (e.g., [rare drop notifier](https://runelite.net/plugin-hub/show/discord-rare-drop-notificater) also supports allowlist)
2. Write custom webhook handler code that forwards your loot notifications to the respective webhooks depending on the value/allowlist (can be a localhost webserver or you can use cloudflare workers / aws lambda to host this for free)

**Comment by asmith-personal** [comment]
Bumping this one, is there not a way for the plugin to parse comma-separated webhook URLs without the need for a custom UI panel?

**Comment by pajlada** [comment]
> Bumping this one, is there not a way for the plugin to parse comma-separated webhook URLs without the need for a custom UI panel?
> 
> EDIT: It seems like you can separate by newline

Yeah, so you can send the same notification to multiple webhooks, so the feature request is for having separate configurations for those notifications (e.g. I want to send every single boss KC to one webhook, but only every 10 boss KC to the other webhook)

**Comment by KosmicAnosmic** [comment]
Adding onto this since it seems like a similar request, but is there any way a second Custom Chat Messages tab could be created with a similar intent?

I'd like to separate my Dink commands (::dink *) from actual in game notifications (like a clan member playing on mobile getting a big drop) if possible.
